src/providers/MeetingProvider/docs/MeetingProvider.stories.mdx

```diff
@@ -105,4 +105,4 @@ variable values and you will not use video, roster and audio at both places. Sti
 The video will not work due to limitations of JS SDK to bind a video stream to a single video tile only. Please check [this](https://github.com/aws/amazon-chime-sdk-component-library-react/issues/492#issuecomment-846317339) issue for more details.
 
 Please make sure your app calls `meetingManager.join()` only once with the same `attendeeInfo`.
-Otherwise,  the previous attendee who joined the meeting will leave the meeting with [AudioJoinedFromAnotherDevice](https://aws.github.io/amazon-chime-sdk-js/enums/meetingsessionstatuscode.html#audiojoinedfromanotherdevice).
+Otherwise,  the previous attendee who joined the meeting will leave the meeting with [AudioJoinedFromAnotherDevice](https://aws.github.io/amazon-chime-sdk-js/enums/meetingsessionstatuscode.html#audiojoinedfromanotherdevice).
```

Comment:
Not sure what have changed, but I see a gap between `Otherwise,` and `the previous`. Change it to one space character.

<img width="300" src="https://user-images.githubusercontent.com/36240708/125140131-b0d7a100-e0c6-11eb-8b47-60b7fc946513.png">

Reply:
Yeah not sure, may be I removed the last line. But, removed the extra space.